content/en/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -57,6 +69,41 @@ kubectl apply -f https://raw.githubusercontent.com/kubernetes/website/master/con
 deployment.apps/my-nginx created
 ```
 
+
+### Group similar resources with PodList, ServiceList, and others
+
+Resources can be gathered into a list that is made up only of items of a particular type, then applied as needed. For example, you could create a list of pods in a `PodList`, services in a `ServiceList`, or secrets in a `SecretList`. The following is a list of supported list types:
```

Comment:
No. You cannot create a list of resources in this way.

Reply:
If kubelet is reading the file, I believe it does work this way :/

You can't send these to apiserver via API though.

Reply:
Yes, I was aware that it was a kubelet thing. I should have asked if this was something you all wanted documented before I just jumped in. I believe Helm was doing something similar, but I'm not sure.

Reply:
I removed the content I added to this file.